docs/software-development-kits/java/authorizer.mdx

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_label: Authorizer Client
+title: Aserto SDKs - Java - Authorizer Client
+description: Aserto SDKs - Java - Creating a client and making authorization calls
+---
+
+# Authorizer Client
+
+[`AuthorizerClient`](https://github.com/aserto-dev/aserto-java/blob/98e155e0305d14685256b2bde0d14e4b58f1bc75/src/main/java/com/aserto/AuthorizerClient.java#L13) is an abstraction over the raw grpc bindings that talks to the Aserto [authorization API](/docs/authorizer-guide/authz).
+It can be used to make authorization calls, or to create authorization middleware.
+
+
+## Installation
+The code is available as a [maven component](https://central.sonatype.com/artifact/com.aserto/aserto-java/)
+and can be easily added to you project by adding the maven dependency
+```maven
+<dependency>
+    <groupId>com.aserto</groupId>
+    <artifactId>aserto-java</artifactId>
+    <version>1.0.1</version>
+</dependency>
+```
+
+## Create a Client
+
+```java
+// create a channel that has the connection details
+ManagedChannel channel = new ChannelBuilder()
+        .withAddr("localhost:8282")
+        .withInsecure(true)
```

Comment:
Can we keep the channel builder secure using the CA cert path instead of setting up an insecure connection ?

Reply:
I think we could have two examples - the first using `.withCACertPath`, and another that says "for dev / test scenarios, you can also run `withInsecure(true)`. This way we are leading with the secure approach but giving the user an easy path for testing without having to obtain / trust self-signed certs.

Reply:
I changed the example to use certificates, made a small note on where to get the certificate, and also how to use an insecure connection.